slug.go

```diff
@@ -158,6 +158,13 @@ func (p *Packer) Pack(src string, w io.Writer) (*Meta, error) {
 		ignoreRules = parseIgnoreFile(src)
 	}
 
+	if !filepath.IsAbs(src) {
```

Comment:
By my understanding, this check is redundant because filepath.Abs will return the path if it's already absolute, or else it will make it absolute using the current working directory.